classes/game.rb

```diff
@@ -8,4 +8,33 @@ def initialize(params)
     @multiplayer = params[:multiplayer]
     @last_played_at = Date.parse(params[:last_played_at])
   end
+
+  private
+
+  def can_be_archived?
+    # super && last_played_old_enough?
+    last_played_old_enough?
+  end
```

Comment:
- The unit test require the file where the source code of the class is stored (its own .rb file)
- The line 15 is commented, so `can_be_archived?` method returns just the result of `last_played_old_enough?` method

Please kindly verify the requirements to fix this.

Reply:
Fixed